sample/HttpTrigger-CSharp-UserAuth/run.csx

```diff
@@ -0,0 +1,30 @@
+﻿using System.Linq;
+using System.Net;
+using System.Security.Claims;
+
+public static HttpResponseMessage Run(HttpRequestMessage req, ClaimsPrincipal principal, TraceWriter log)
+{
+    log.Info(string.Format("C# HTTP trigger function processed a request. {0}", req.RequestUri));

```

Comment:
Is this how we normally log in the samples? I was expecting something that used the newer string formatting method:
```csharp
log.Info($"C# HTTP trigger function processed a request. {req.RequestUri}"); 
```

Reply:
I think you are right. I copied the base of this from an older code review. I will change this.

---
In reply to: [176886711](https://github.com/Azure/azure-functions-host/pull/2577#discussion_r176886711) [](ancestors = 176886711)